.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules
-.env.*
\ No newline at end of file
+.env.*
+.env.development
+.env.test
```

Comment:
.env.* will include any .env.<something> files itself so there is no need to then specify the individual files for .env.development and .env.test after this.